pkg/account/model.go

```diff
@@ -90,6 +92,32 @@ func (a *Account) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// UnmarshalDynamoDBAttributeValue handles custom unmarshaling of an ARN
+func (a *Account) UnmarshalDynamoDBAttributeValue(av *dynamodb.AttributeValue) error {
+
+	type Alias Account
```

Comment:
If anyone is wondering why this exists like this.  Its to prevent infinite loops with line 101.

Reply:
Is this `UnmarshalDynamoDBAttributeValue()` method used anywhere? My IDE couldn't find any usages...

Reply:
yes, its an interface used by dynamodbattribute.Unmarshal